navigation-timing/nav2_test_attributes_exist.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>window.performance.getEntriesByType("navigation") attributes</title>
+        <link rel="author" title="Google" href="http://www.google.com/" />
+        <link rel="help" href="http://www.w3.org/TR/navigation-timing-2/#sec-PerformanceNavigationTiming"/>
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+    </head>
+    <body>
+        <h1>Description</h1>
+        <p>This test validates that PerformanceObserver can observe nav timing 2 instance and that the expected attributes in nav timing 2 instance exist (but does not validate that the values are correct).</p>
+
+        <script>
+        var navTiming2Attributes = [
+            'connectEnd',
+            'connectStart',
+            'decodedBodySize',
+            'domComplete',
+            'domContentLoadedEventEnd',
+            'domContentLoadedEventStart',
+            'domInteractive',
+            'domainLookupEnd',
+            'domainLookupStart',
+            'duration',
+            'encodedBodySize',
+            'entryType',
+            'fetchStart',
+            'initiatorType',
+            'loadEventEnd',
+            'loadEventStart',
+            'name',
+            'redirectCount',
+            'redirectEnd',
+            'redirectStart',
+            'requestStart',
+            'responseEnd',
+            'responseStart',
+            'secureConnectionStart',
+            'unloadEventEnd',
+            'unloadEventStart',
+            'workerStart'
```

Comment:
- transferSize is missing?
- ditto for nextHopProtocol

Nitpick: it would much easier to read/review if these were sorted in ~rough order of how they're recorded or shown in our processing diagram: http://w3c.github.io/navigation-timing/#processing-model